conf/supervisord.conf

```
@@ -12,21 +12,23 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 
 [program:web]
-environment=NEW_RELIC_CONFIG_FILE=conf/app.ini
+NEW_RELIC_CONFIG_FILE=conf/newrelic-web.ini
```

I'm not sure you can just put environment variables in here like this. I think you need to use `environment=` with key value pairs: http://supervisord.org/configuration.html#supervisord-section-values

Also, as we only want to set one thing, can we get it all done here?
```suggestion
environment=NEW_RELIC_APP_NAME=h-web
```

---

Thanks Jon - Updates have been made and pushed.